sdk/keyvault/Azure.Security.KeyVault.Shared/src/KeyVaultPipeline.cs

```diff
@@ -69,12 +72,14 @@ public Request CreateRequest(RequestMethod method, Uri uri, bool appendApiVersio
                 request.Uri.AppendQuery("api-version", ApiVersion);
             }
 
-            return request;
+            return message;
         }
 
-        public Request CreateRequest(RequestMethod method, params string[] path)
+        public HttpMessage CreateMessage(RequestMethod method, RequestContext context, params string[] path)
         {
-            Request request = _pipeline.CreateRequest();
+            // See comment in CreateMessage overload above for future design consideration.
+            HttpMessage message = _pipeline.CreateMessage(context);
```

Comment:
The expectation is that the generated code will pass in a StatusCodeClassifier with a list of all the valid non-error status code, which it has obtained from the API spec.  Since you are writing this manually, you will need to do this by hand in order to satisfy the contract.  We can discuss whether this is a good design for hand-written code, but the APIs were not designed to work without a DPG client, thus the need to work within the spec on this.

```suggestion
            HttpMessage message = _pipeline.CreateMessage(context, new StatusCodeClassifier(stackalloc ushort[] { 200 }));
```

Reply:
If the `ResponseClassifier` parameter is required as you suggest, why is the parameter optional via overload? Seems it shouldn't be an overload if I'm allowed to call it "wrong".

/cc @m-nash

Reply:
These APIs were designed to be called by generated code and are reasonably advanced.  I believe, though, that nothing should block you from making the same calls the generated code does manually.